openrtb_ext/request.go

```diff
@@ -151,14 +151,15 @@ type ExtRequestPrebidCacheVAST struct {
 
 // ExtRequestTargeting defines the contract for bidrequest.ext.prebid.targeting
 type ExtRequestTargeting struct {
-	PriceGranularity     *PriceGranularity        `json:"pricegranularity,omitempty"`
-	IncludeWinners       *bool                    `json:"includewinners,omitempty"`
-	IncludeBidderKeys    *bool                    `json:"includebidderkeys,omitempty"`
-	IncludeBrandCategory *ExtIncludeBrandCategory `json:"includebrandcategory,omitempty"`
-	IncludeFormat        bool                     `json:"includeformat,omitempty"`
-	DurationRangeSec     []int                    `json:"durationrangesec,omitempty"`
-	PreferDeals          bool                     `json:"preferdeals,omitempty"`
-	AppendBidderNames    bool                     `json:"appendbiddernames,omitempty"`
+	PriceGranularity          *PriceGranularity          `json:"pricegranularity,omitempty"`
+	IncludeWinners            *bool                      `json:"includewinners,omitempty"`
+	IncludeBidderKeys         *bool                      `json:"includebidderkeys,omitempty"`
+	IncludeBrandCategory      *ExtIncludeBrandCategory   `json:"includebrandcategory,omitempty"`
+	IncludeFormat             bool                       `json:"includeformat,omitempty"`
+	DurationRangeSec          []int                      `json:"durationrangesec,omitempty"`
+	PreferDeals               bool                       `json:"preferdeals,omitempty"`
+	AppendBidderNames         bool                       `json:"appendbiddernames,omitempty"`
+	MediaTypePriceGranularity *MediaTypePriceGranularity `json:"mediatypepricegranularity,omitempty"`
```

Comment:
Nitpick: I liked how you had the `MediaTypePriceGranularity` come right after `PriceGranularity`, could you move the media type one to right under line 154?